thorlcr/master/thgraphmanager.cpp

```diff
@@ -1001,12 +1002,52 @@ bool CJobManager::executeGraph(IConstWorkUnit &workunit, const char *graphName,
         } cBlock(activeTasks);
 
         {
+#ifdef _CONTAINERIZED
+            // collate pod distribution
+            VStringBuffer selector("thorworker-%s-%s", wuid.get(), graphName);
+            std::vector<std::vector<std::string>> pods = getPodNodes(selector.toLowerCase());
+            std::unordered_map<std::string, unsigned> podPerNodeCounts;
+            for (const auto &podNode: pods)
+            {
+                const std::string &node = podNode[1]; // pod is 1st item, node is 2nd
+                podPerNodeCounts[node]++; // NB: if doesn't exist is created with default value of 0 1st
+            }
+            const StatisticsMapping podStatistics({StNumPods});
+            CRuntimeSummaryStatisticCollection podStats(podStatistics);
+            unsigned nodes=0;
+            double sum=0;
+            double sumSquares=0;
+            for (auto &node: podPerNodeCounts)
+            {
+                podStats.setStatistic(StNumPods, node.second, nodes++);
+                sum += node.second;
+                sumSquares += node.second*node.second;
+            }
+            double mean = sum / nodes;
+            double variance = (sumSquares - sum * mean) / nodes;
+            double stdDev = sqrt(variance);
```

Comment:
sum and sumSquares are already calculated in podStats.  
Cleaner to add a CRuntimeSummaryStatisticCollection::queryStdDev(kind) function:
{ derived[queryMapping().getIndex(kind)]->queryStdDev() }